Changes

```diff
@@ -266,6 +266,9 @@ OCaml 5.2.0
   (Gabriel Scherer and Damien Doligez, review by Daniel Bünzli, David
    Allsopp, Andreas Rossberg, Xavier Leroy, report by Andreas Rossberg)
 
+- #13130: minor fixes to pprintast for raw identifiers and local module open
+  syntax for types.
```

Comment:
You should add a credit line, probably something like "(Chet Murthy, review by Gabriel Scherer)" (on its own line as for other entries.)